### PR TITLE
[infra] bumped mailgun.api

### PR DESCRIPTION
**Pull request by lappkast**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -21,6 +21,3 @@ PLATFORMS
 
 DEPENDENCIES
   albacore (>= 2.3.6)
-
```
Comment (haf) [inline Gemfile.lock]: Please do `gem update bundler`

**Comment by haf** [comment]
If you remove the unneeded change to Gemfile.lock, I can merge.

**Comment by haf** [comment]
I merged anyway ;)

**Comment by lappkast** [comment]
:)

**Comment by haf** [comment]
This merge broke the build.

**Comment by lappkast** [comment]
Checking

**Comment by haf** [comment]
I think I've solved it.

**Comment by lappkast** [comment]
Oops. Want another pr?

**Comment by haf** [comment]
All done now. Released changes.

**Comment by haf** [comment]
If you want you could test these changes in our app, by triggering a JS error – SuaveReporter has changed from using `Aether.idLens <-?> Json.ObjectPIso` to using `Aether.id_ >-?> Json.Object_` i.e. a change from a partially isomorphic function to a partial lens.

**Comment by lappkast** [comment]
Looks the same to me:
id_ <-?> Json.Object__

**Comment by lappkast** [comment]
I think this: idLens <-?> Json.StringPIso
Should become this: Json.String_
I'm sending another pr.

**Pull request merged**